### PR TITLE
Do Place spaces between control statements and their parentheses.

### PR DESCRIPTION
**Pull request by schnek**
**Todo / Checklist**
- [x] Target is branch develop.
- [x] Detailed description about this pull request.
- [x] Checked AE-Coding standards.

**Tests Performed:** 
- [x] Build AE.
- [x] Server startup.
- [x] Log into world.

**Multiversion Ingame Tests Performed:**
- [x] BC
- [x] WotLK
- [x] Cata

clean cmake cache and rebuild the source files

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -8,15 +8,15 @@ cmake_policy(SET CMP0057 NEW)
...
         message(STATUS "Build configuration was not detected, setting to \"Release\"")
         set(CMAKE_BUILD_TYPE "Release")
     else()
         message(STATUS "Detected ${CMAKE_BUILD_TYPE} configuration")
     endif()
 
-    if(CMAKE_SYSTEM_NAME STREQUAL "FreeBSD" OR CMAKE_SYSTEM_NAME STREQUAL "kFreeBSD")
+    if (CMAKE_SYSTEM_NAME STREQUAL "FreeBSD" OR CMAKE_SYSTEM_NAME STREQUAL "kFreeBSD")
         set(CMAKE_CXX_LINK_FLAGS "${CMAKE_CXX_LINK_FLAGS} -lc++experimental")
     elseif(APPLE)
```
Comment (Zyres) [inline CMakeLists.txt:21]: Missing

File: cmake/CheckSystem.cmake [hunk 2]
```diff
@@ -42,10 +42,10 @@ mark_as_advanced(
 )
 
 # apply system settings
-if(WIN32)
+if (WIN32)
     include(${CMAKE_SOURCE_DIR}/cmake/Systems/Windows.cmake)
 elseif(UNIX)
```
Comment (Zyres) [inline cmake/CheckSystem.cmake:47]: Missing

File: cmake/CheckSystem.cmake [hunk 3]
```diff
@@ -42,10 +42,10 @@ mark_as_advanced(
...
-    if(APPLE)
+    if (APPLE)
         include(${CMAKE_SOURCE_DIR}/cmake/Systems/Apple.cmake) 
     elseif(CMAKE_SYSTEM_NAME STREQUAL "FreeBSD" OR CMAKE_SYSTEM_NAME STREQUAL "kFreeBSD")
```
Comment (Zyres) [inline cmake/CheckSystem.cmake:50]: Missing

File: cmake/Systems/Apple.cmake [hunk 4]
```diff
@@ -14,7 +14,7 @@ find_package(Threads REQUIRED)
 find_package(MySQL REQUIRED)
 find_package(BZip2 REQUIRED)
 
-if(CMAKE_COMPILER_IS_GNUCXX)
+if (CMAKE_COMPILER_IS_GNUCXX)
     include(${CMAKE_SOURCE_DIR}/cmake/Compilers/gcc.cmake)
 elseif(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
```
Comment (Zyres) [inline cmake/Systems/Apple.cmake:19]: Missing

File: cmake/Systems/Linux.cmake [hunk 5]
```diff
@@ -15,7 +15,7 @@ find_package(Threads REQUIRED)
 find_package(MySQL REQUIRED)
 find_package(BZip2 REQUIRED)
 
-if(CMAKE_COMPILER_IS_GNUCXX)
+if (CMAKE_COMPILER_IS_GNUCXX)
     include(${CMAKE_SOURCE_DIR}/cmake/Compilers/gcc.cmake)
 elseif(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
```
Comment (Zyres) [inline cmake/Systems/Linux.cmake:20]: Missing

File: dep/CMakeLists.txt [hunk 6]
```diff
@@ -21,11 +21,11 @@ if (NOT WIN32)
...
 endif()
 
-if(BUILD_ASCEMUSCRIPTS AND BUILD_LUAENGINE)
+if (BUILD_ASCEMUSCRIPTS AND BUILD_LUAENGINE)
 	add_subdirectory(lualib/src)
 endif()
 
-if(BUILD_TOOLS)
+if (BUILD_TOOLS)
     if("${ASCEMU_VERSION}" STREQUAL "Cata" OR "${ASCEMU_VERSION}" STREQUAL "Mop")
```
Comment (Zyres) [inline dep/CMakeLists.txt:29]: Missing

File: src/scripts/ExtraScripts/CMakeLists.txt [hunk 7]
```diff
@@ -1,14 +1,14 @@
 # Copyright (c) 2014-2021 AscEmu Team <http://www.ascemu.org>
 
-if(NOT BUILD_EXTRASCRIPTS )
+if (NOT BUILD_EXTRASCRIPTS )
   message("\tExtraScripts was skipped because BUILD_EXTRASCRIPTS is unchecked.")
   return()
 endif(NOT BUILD_EXTRASCRIPTS)
```
Comment (Zyres) [inline src/scripts/ExtraScripts/CMakeLists.txt:6]: Missing

File: src/scripts/GossipScripts/CMakeLists.txt [hunk 8]
```diff
@@ -1,14 +1,14 @@
 # Copyright (c) 2014-2021 AscEmu Team <http://www.ascemu.org>
 
-if(NOT BUILD_GOSSIPSCRIPTS )
+if (NOT BUILD_GOSSIPSCRIPTS )
   message("\tGossipScripts was skipped because BUILD_GOSSIPSCRIPTS is unchecked.")
   return()
 endif(NOT BUILD_GOSSIPSCRIPTS)
```
Comment (Zyres) [inline src/scripts/GossipScripts/CMakeLists.txt:6]: Missing

File: src/shared/Network/CMakeLists.txt [hunk 9]
```diff
@@ -19,7 +19,7 @@ set(SRC_NETWORK_FILES
    ${PATH_PREFIX}/SocketOps.h
 )
 
-if(WIN32)
+if (WIN32)
    list(APPEND SRC_NETWORK_FILES ${sources} ${PATH_PREFIX}/SocketMgrWin32.cpp ${PATH_PREFIX}/SocketOpsWin32.cpp ${PATH_PREFIX}/SocketWin32.cpp)
 elseif((CMAKE_SYSTEM_NAME STREQUAL "FreeBSD" OR CMAKE_SYSTEM_NAME STREQUAL "kFreeBSD") OR APPLE)
```
Comment (Zyres) [inline src/shared/Network/CMakeLists.txt:24]: Missing

File: src/tools/CMakeLists.txt [hunk 10]
```diff
@@ -1,7 +1,7 @@
 # Copyright (c) 2014-2021 AscEmu Team <http://www.ascemu.org>
 
-#check which tools are set
-if(BUILD_TOOLS)
+# check which tools are set
+if (BUILD_TOOLS)
    if("${ASCEMU_VERSION}" STREQUAL "WotLK" OR "${ASCEMU_VERSION}" STREQUAL "TBC" OR "${ASCEMU_VERSION}" STREQUAL "Classic")
```
Comment (Zyres) [inline src/tools/CMakeLists.txt:5]: Missing

**Review by Zyres — CHANGES_REQUESTED** [review]
Please check out the missing spaces for elseif too.
Take your time, we should solve the missing spaces in one PR

File: CMakeLists.txt [hunk 11]
```diff
@@ -8,31 +8,31 @@ cmake_policy(SET CMP0057 NEW)
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 
 # set build type on unix if it wasn't defined by user
-if(UNIX)
-    if(NOT CMAKE_BUILD_TYPE)
+if (UNIX)
+    if (NOT CMAKE_BUILD_TYPE)
         message(STATUS "Build configuration was not detected, setting to \"Release\"")
         set(CMAKE_BUILD_TYPE "Release")
     else()
```
Comment (Zyres) [inline CMakeLists.txt:15]: now else is also missing....

File: cmake/CheckSystem.cmake [hunk 12]
```diff
@@ -9,23 +9,23 @@ list(APPEND CMAKE_MODULE_PATH
 include(${CMAKE_SOURCE_DIR}/cmake/GitRevision.cmake)
 
 # get architecture type
-if(CMAKE_SIZEOF_VOID_P EQUAL 8)
+if (CMAKE_SIZEOF_VOID_P EQUAL 8)
     set(IS_64BIT TRUE)
 else()
```
Comment (Zyres) [inline cmake/CheckSystem.cmake:14]: missing

File: cmake/CheckSystem.cmake [hunk 13]
```diff
@@ -9,23 +9,23 @@ list(APPEND CMAKE_MODULE_PATH
...
-endif()
+endif ()
 
 # set default architecture identifier
-if(IS_64BIT)
+if (IS_64BIT)
     message(STATUS "Detected x64 system")
-	message(STATUS "Generator Plattform: ${CMAKE_GENERATOR_PLATFORM}")
+    message(STATUS "Generator Plattform: ${CMAKE_GENERATOR_PLATFORM}")
 else()
```
Comment (Zyres) [inline cmake/CheckSystem.cmake:22]: missing

File: cmake/CheckSystem.cmake [hunk 14]
```diff
@@ -42,21 +42,21 @@ mark_as_advanced(
...
+elseif (UNIX)
+    if (APPLE)
         include(${CMAKE_SOURCE_DIR}/cmake/Systems/Apple.cmake) 
-    elseif(CMAKE_SYSTEM_NAME STREQUAL "FreeBSD" OR CMAKE_SYSTEM_NAME STREQUAL "kFreeBSD") 
+    elseif (CMAKE_SYSTEM_NAME STREQUAL "FreeBSD" OR CMAKE_SYSTEM_NAME STREQUAL "kFreeBSD") 
         include(${CMAKE_SOURCE_DIR}/cmake/Systems/FreeBSD.cmake) 
-    elseif(${CMAKE_SYSTEM_NAME} MATCHES "Linux") 
+    elseif (${CMAKE_SYSTEM_NAME} MATCHES "Linux") 
         include(${CMAKE_SOURCE_DIR}/cmake/Systems/Linux.cmake) 
     else()
```
Comment (Zyres) [inline cmake/CheckSystem.cmake:54]: missing

File: cmake/Compilers/clang.cmake [hunk 15]
```diff
@@ -17,12 +17,12 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O2 -std=c++17")
 if (IS_64BIT)
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fPIC")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")
-endif()
+endif ()
 
 if (BUILD_WITH_WARNINGS)
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Wextra")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra")
 else()
```
Comment (Zyres) [inline cmake/Compilers/clang.cmake:25]: missing

File: cmake/Compilers/gcc.cmake [hunk 16]
```diff
@@ -17,12 +17,12 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++17 -O2")
 if (IS_64BIT)
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fPIC")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")
-endif()
+endif ()
 
 if (BUILD_WITH_WARNINGS)
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Wextra")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra")
 else()
```
Comment (Zyres) [inline cmake/Compilers/gcc.cmake:25]: missing

File: cmake/Compilers/msvc.cmake [hunk 17]
```diff
@@ -30,4 +30,4 @@ if (BUILD_WITH_WARNINGS)
 else()
```
Comment (Zyres) [inline cmake/Compilers/msvc.cmake:30]: missing

File: cmake/Modules/FindMySQL.cmake [hunk 18]
```diff
@@ -6,20 +6,20 @@
...
             "$ENV{ProgramFiles\(x86\)}/MySQL/*/include"
-	        "$ENV{ProgramFiles}/MySQL/*/include"
+            "$ENV{ProgramFiles}/MySQL/*/include"
             "$ENV{ProgramW6432}/MySQL/*/include"
             "$ENV{SystemDrive}/MySQL/*/include"
             "/usr/include/mysql"
             "/usr/local/include/mysql"
             "/usr/mysql/include/mysql")
-else ()
+else()
```
Comment (Zyres) [inline cmake/Modules/FindMySQL.cmake:20]: removed, why?

File: cmake/Modules/FindMySQL.cmake [hunk 19]
```diff
@@ -30,18 +30,18 @@ endif ()
...
-if(NOT IS_64BIT)
+if (NOT IS_64BIT)
 LIST(APPEND MY_LIB_PATHS 
             "$ENV{ProgramFiles\(x86\)}/MySQL/*/lib"
-	        "$ENV{ProgramFiles}/MySQL/*/lib"
+            "$ENV{ProgramFiles}/MySQL/*/lib"
             "$ENV{ProgramW6432}/MySQL/*/lib"
             "$ENV{SystemDrive}/MySQL/*/lib")
-else ()
+else()
```
Comment (Zyres) [inline cmake/Modules/FindMySQL.cmake:42]: removed why?

File: cmake/Modules/FindMySQL.cmake [hunk 20]
```diff
@@ -50,15 +50,15 @@ find_library(MYSQL_LIBRARY
     NAMES "libmysql"
     PATHS ${MY_LIB_PATHS})
 
-else (WIN32)
+else(WIN32)
```
Comment (Zyres) [inline cmake/Modules/FindMySQL.cmake:53]: removed why

File: cmake/Modules/FindMySQL.cmake [hunk 21]
```diff
@@ -72,10 +72,10 @@ if (MYSQL_LIBRARY)
             STRING(REPLACE ".lib" ".dll" MYSQL_DLL ${MYSQL_DLL})
             message(STATUS "Found MySQL dll: ${MYSQL_DLL}")
         endif (WIN32)
-    else (MYSQL_INCLUDE_DIR)
+    else(MYSQL_INCLUDE_DIR)
```
Comment (Zyres) [inline cmake/Modules/FindMySQL.cmake:75]: removed why?

File: cmake/Modules/FindMySQL.cmake [hunk 22]
```diff
@@ -72,10 +72,10 @@ if (MYSQL_LIBRARY)
...
         message(FATAL_ERROR "Could not find MySQL headers! Please install the development libraries and headers")
     endif (MYSQL_INCLUDE_DIR)
     mark_as_advanced(MYSQL_FOUND MYSQL_LIBRARY)
-else (MYSQL_LIBRARY)
+else(MYSQL_LIBRARY)
```
Comment (Zyres) [inline cmake/Modules/FindMySQL.cmake:79]: removed why?

File: cmake/Modules/FindOpenSSL.cmake [hunk 23]
```diff
@@ -83,29 +83,29 @@ set(OPENSSL_EXPECTED_VERSION "1.0")
 set(OPENSSL_MAX_VERSION "1.2")
 
 macro(_OpenSSL_test_and_find_dependencies ssl_library crypto_library)
-  if((CMAKE_SYSTEM_NAME STREQUAL "Linux") AND
+  if ((CMAKE_SYSTEM_NAME STREQUAL "Linux") AND
      (("${ssl_library}" MATCHES "\\${CMAKE_STATIC_LIBRARY_SUFFIX}$") OR
       ("${crypto_library}" MATCHES "\\${CMAKE_STATIC_LIBRARY_SUFFIX}$")))
     set(_OpenSSL_has_dependencies TRUE)
     find_package(Threads)
   else()
```
Comment (Zyres) [inline cmake/Modules/FindOpenSSL.cmake:91]: missing

File: cmake/Modules/FindOpenSSL.cmake [hunk 24]
```diff
@@ -114,19 +114,19 @@ if (UNIX)
 endif ()
 
 # Support preference of static libs by adjusting CMAKE_FIND_LIBRARY_SUFFIXES
-if(OPENSSL_USE_STATIC_LIBS)
+if (OPENSSL_USE_STATIC_LIBS)
   set(_openssl_ORIG_CMAKE_FIND_LIBRARY_SUFFIXES ${CMAKE_FIND_LIBRARY_SUFFIXES})
-  if(WIN32)
+  if (WIN32)
     set(CMAKE_FIND_LIBRARY_SUFFIXES .lib .a ${CMAKE_FIND_LIBRARY_SUFFIXES})
   else()
```
Comment (Zyres) [inline cmake/Modules/FindOpenSSL.cmake:121]: missing

File: cmake/Modules/FindOpenSSL.cmake [hunk 25]
```diff
@@ -154,15 +154,15 @@ if (WIN32)
...
   # If OpenSSL was installed using .msi package instead of .exe, Inno Setup registry values are not written to Uninstall\OpenSSL
   # but because it is only a shim around Inno Setup it does write the location of uninstaller which we can use to determine path
   get_filename_component(_OPENSSL_MSI_INSTALL_PATH "[HKEY_LOCAL_MACHINE\\SOFTWARE\\Inno Setup MSIs\\${_OPENSSL_MSI_INSTALL_GUID};]" DIRECTORY)
-  if(NOT _OPENSSL_MSI_INSTALL_PATH STREQUAL "/")
+  if (NOT _OPENSSL_MSI_INSTALL_PATH STREQUAL "/")
     list(INSERT _OPENSSL_ROOT_HINTS 0 ${_OPENSSL_MSI_INSTALL_PATH})
-  endif()
-else ()
+  endif ()
+else()
```
Comment (Zyres) [inline cmake/Modules/FindOpenSSL.cmake:165]: removed why?

File: cmake/Modules/FindOpenSSL.cmake [hunk 26]
```diff
@@ -205,18 +205,18 @@ if(WIN32 AND NOT CYGWIN)
 
     if (OPENSSL_MSVC_STATIC_RT)
       set(_OPENSSL_MSVC_RT_MODE "MT")
-    else ()
+    else()
```
Comment (Zyres) [inline cmake/Modules/FindOpenSSL.cmake:208]: removed why?

File: cmake/Modules/FindOpenSSL.cmake [hunk 27]
```diff
@@ -205,18 +205,18 @@ if(WIN32 AND NOT CYGWIN)
...
       set(_OPENSSL_MSVC_RT_MODE "MD")
     endif ()
 
     # Since OpenSSL 1.1, lib names are like libcrypto32MTd.lib and libssl32MTd.lib
-    if( "${CMAKE_SIZEOF_VOID_P}" STREQUAL "8" )
+    if ( "${CMAKE_SIZEOF_VOID_P}" STREQUAL "8" )
         set(_OPENSSL_MSVC_ARCH_SUFFIX "64")
     else()
```
Comment (Zyres) [inline cmake/Modules/FindOpenSSL.cmake:215]: missing

File: cmake/Modules/FindOpenSSL.cmake [hunk 28]
```diff
@@ -471,19 +471,19 @@ function(from_hex HEX DEC)
       math(EXPR _res "${_res} + 15")
     else()
```
Comment (Zyres) [inline cmake/Modules/FindOpenSSL.cmake:472]: missing

File: cmake/Modules/FindOpenSSL.cmake [hunk 29]
```diff
@@ -512,7 +512,7 @@ if(OPENSSL_INCLUDE_DIR AND EXISTS "${OPENSSL_INCLUDE_DIR}/openssl/opensslv.h")
     endif ()
 
     set(OPENSSL_VERSION "${OPENSSL_VERSION_MAJOR}.${OPENSSL_VERSION_MINOR}.${OPENSSL_VERSION_FIX}${OPENSSL_VERSION_PATCH_STRING}")
-  else ()
+  else()
```
Comment (Zyres) [inline cmake/Modules/FindOpenSSL.cmake:515]: removed why?

File: cmake/Modules/FindOpenSSL.cmake [hunk 30]
```diff
@@ -527,30 +527,30 @@ if(OPENSSL_INCLUDE_DIR AND EXISTS "${OPENSSL_INCLUDE_DIR}/openssl/opensslv.h")
...
-  if(_comp STREQUAL "Crypto")
-    if(EXISTS "${OPENSSL_INCLUDE_DIR}" AND
+  if (_comp STREQUAL "Crypto")
+    if (EXISTS "${OPENSSL_INCLUDE_DIR}" AND
         (EXISTS "${OPENSSL_CRYPTO_LIBRARY}" OR
         EXISTS "${LIB_EAY_LIBRARY_DEBUG}" OR
         EXISTS "${LIB_EAY_LIBRARY_RELEASE}")
     )
       set(OpenSSL_${_comp}_FOUND TRUE)
     else()
```
Comment (Zyres) [inline cmake/Modules/FindOpenSSL.cmake:537]: missing

File: cmake/Modules/FindOpenSSL.cmake [hunk 31]
```diff
@@ -527,30 +527,30 @@ if(OPENSSL_INCLUDE_DIR AND EXISTS "${OPENSSL_INCLUDE_DIR}/openssl/opensslv.h")
...
-    if(EXISTS "${OPENSSL_INCLUDE_DIR}" AND
+    endif ()
+  elseif (_comp STREQUAL "SSL")
+    if (EXISTS "${OPENSSL_INCLUDE_DIR}" AND
         (EXISTS "${OPENSSL_SSL_LIBRARY}" OR
         EXISTS "${SSL_EAY_LIBRARY_DEBUG}" OR
         EXISTS "${SSL_EAY_LIBRARY_RELEASE}")
     )
       set(OpenSSL_${_comp}_FOUND TRUE)
     else()
```
Comment (Zyres) [inline cmake/Modules/FindOpenSSL.cmake:547]: missing

File: cmake/Modules/FindOpenSSL.cmake [hunk 32]
```diff
@@ -527,30 +527,30 @@ if(OPENSSL_INCLUDE_DIR AND EXISTS "${OPENSSL_INCLUDE_DIR}/openssl/opensslv.h")
...
       set(OpenSSL_${_comp}_FOUND FALSE)
-    endif()
+    endif ()
   else()
```
Comment (Zyres) [inline cmake/Modules/FindOpenSSL.cmake:550]: missing

File: cmake/Modules/FindPCRE.cmake [hunk 33]
```diff
@@ -5,27 +5,27 @@
...
-  SET( PCRE_LIBRARIES ${PCRE_LIBRARY} )
-  SET( PCRE_INCLUDE_DIRS ${PCRE_INCLUDE_DIR} )
-ELSE(PCRE_FOUND)
-  SET( PCRE_LIBRARIES )
-  SET( PCRE_INCLUDE_DIRS )
-ENDIF(PCRE_FOUND)
+if (PCRE_FOUND)
+    set( PCRE_LIBRARIES ${PCRE_LIBRARY} )
+    set( PCRE_INCLUDE_DIRS ${PCRE_INCLUDE_DIR} )
+else(PCRE_FOUND)
```
Comment (Zyres) [inline cmake/Modules/FindPCRE.cmake:26]: missing

File: cmake/Systems/Apple.cmake [hunk 34]
```diff
@@ -14,14 +14,14 @@ find_package(Threads REQUIRED)
 find_package(MySQL REQUIRED)
 find_package(BZip2 REQUIRED)
 
-if(CMAKE_COMPILER_IS_GNUCXX)
+if (CMAKE_COMPILER_IS_GNUCXX)
     include(${CMAKE_SOURCE_DIR}/cmake/Compilers/gcc.cmake)
-elseif(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
+elseif (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
     include(${CMAKE_SOURCE_DIR}/cmake/Compilers/clang.cmake)
 else()
```
Comment (Zyres) [inline cmake/Systems/Apple.cmake:21]: missing

File: cmake/Systems/FreeBSD.cmake [hunk 35]
```diff
@@ -12,14 +12,14 @@ find_package(Threads REQUIRED)
 find_package(MySQL REQUIRED)
 find_package(BZip2 REQUIRED)
 
-if(CMAKE_COMPILER_IS_GNUCXX)
+if (CMAKE_COMPILER_IS_GNUCXX)
     include(${CMAKE_SOURCE_DIR}/cmake/Compilers/gcc.cmake)
-elseif(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
+elseif (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
     include(${CMAKE_SOURCE_DIR}/cmake/Compilers/clang.cmake)
 else()
```
Comment (Zyres) [inline cmake/Systems/FreeBSD.cmake:19]: missing

File: cmake/Systems/Linux.cmake [hunk 36]
```diff
@@ -15,14 +15,14 @@ find_package(Threads REQUIRED)
 find_package(MySQL REQUIRED)
 find_package(BZip2 REQUIRED)
 
-if(CMAKE_COMPILER_IS_GNUCXX)
+if (CMAKE_COMPILER_IS_GNUCXX)
     include(${CMAKE_SOURCE_DIR}/cmake/Compilers/gcc.cmake)
-elseif(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
+elseif (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
     include(${CMAKE_SOURCE_DIR}/cmake/Compilers/clang.cmake)
 else()
```
Comment (Zyres) [inline cmake/Systems/Linux.cmake:22]: missing

File: cmake/Systems/Windows.cmake [hunk 37]
```diff
@@ -29,8 +29,8 @@ set(PATH_DB_FILES ${CMAKE_SOURCE_DIR}/sql/)
 
 set(INSTALL_DB_FILES ${PATH_DB_FILES})
 
-if(MSVC)
+if (MSVC)
     include(${CMAKE_SOURCE_DIR}/cmake/Compilers/msvc.cmake)
 else()
```
Comment (Zyres) [inline cmake/Systems/Windows.cmake:34]: missing

File: dep/CMakeLists.txt [hunk 38]
```diff
@@ -1,36 +1,37 @@
 # Copyright (c) 2014-2021 AscEmu Team <http://www.ascemu.org>
 
 #ignore the warnings for dependencies (external code)
-if(MSVC)
+if (MSVC)
     add_definitions(/W0)
 else()
```
Comment (Zyres) [inline dep/CMakeLists.txt:6]: missing

File: dep/CMakeLists.txt [hunk 39]
```diff
@@ -1,36 +1,37 @@
...
 	add_subdirectory(lualib/src)
-endif()
+endif ()
 
-if(BUILD_TOOLS)
-    if("${ASCEMU_VERSION}" STREQUAL "Cata" OR "${ASCEMU_VERSION}" STREQUAL "Mop")
+if (BUILD_TOOLS)
+    if ("${ASCEMU_VERSION}" STREQUAL "Cata" OR "${ASCEMU_VERSION}" STREQUAL "Mop")
         add_subdirectory(StormLib)
     else()
```
Comment (Zyres) [inline dep/CMakeLists.txt:32]: missing

File: src/scripts/InstanceScripts/Cata/CMakeLists.txt [hunk 40]
```diff
@@ -39,7 +39,7 @@ set(SRC_CATA_SCRIPTS_FILES
     ${PATH_PREFIX}/Raid_DragonSoul.h
     ${PATH_PREFIX}/Raid_Firelands.h
 )
-endif()
+endif ()
 
 source_group(Cata FILES ${SRC_CATA_SCRIPTS_FILES})
 unset(PATH_PREFIX)
```
Comment (Zyres) [inline src/scripts/InstanceScripts/Cata/CMakeLists.txt:45]: missing new line

File: src/shared/Network/CMakeLists.txt [hunk 41]
```diff
@@ -3,29 +3,29 @@
...
 
-if(WIN32)
-   list(APPEND SRC_NETWORK_FILES ${sources} ${PATH_PREFIX}/SocketMgrWin32.cpp ${PATH_PREFIX}/SocketOpsWin32.cpp ${PATH_PREFIX}/SocketWin32.cpp)
-elseif((CMAKE_SYSTEM_NAME STREQUAL "FreeBSD" OR CMAKE_SYSTEM_NAME STREQUAL "kFreeBSD") OR APPLE)
-   list(APPEND SRC_NETWORK_FILES ${sources} ${PATH_PREFIX}/SocketMgrFreeBSD.cpp ${PATH_PREFIX}/SocketFreeBSD.cpp ${PATH_PREFIX}/SocketOpsFreeBSD.cpp)
+if (WIN32)
+    list(APPEND SRC_NETWORK_FILES ${sources} ${PATH_PREFIX}/SocketMgrWin32.cpp ${PATH_PREFIX}/SocketOpsWin32.cpp ${PATH_PREFIX}/SocketWin32.cpp)
+elseif ((CMAKE_SYSTEM_NAME STREQUAL "FreeBSD" OR CMAKE_SYSTEM_NAME STREQUAL "kFreeBSD") OR APPLE)
+    list(APPEND SRC_NETWORK_FILES ${sources} ${PATH_PREFIX}/SocketMgrFreeBSD.cpp ${PATH_PREFIX}/SocketFreeBSD.cpp ${PATH_PREFIX}/SocketOpsFreeBSD.cpp)
 else()
```
Comment (Zyres) [inline src/shared/Network/CMakeLists.txt:26]: missing

File: src/tools/CMakeLists.txt [hunk 42]
```diff
@@ -1,20 +1,20 @@
...
-      add_subdirectory(ToolsCataMop/mmaps_generator)
-   endif()
-endif()
+# check which tools are set
+if (BUILD_TOOLS)
+    if ("${ASCEMU_VERSION}" STREQUAL "WotLK" OR "${ASCEMU_VERSION}" STREQUAL "TBC" OR "${ASCEMU_VERSION}" STREQUAL "Classic")
+        add_subdirectory(map_extractor)
+        add_subdirectory(vmap_tools)
+        add_subdirectory(mmaps_generator)
+    else()
```
Comment (Zyres) [inline src/tools/CMakeLists.txt:9]: missing

File: src/tools/CMakeLists.txt [hunk 43]
```diff
@@ -1,20 +1,20 @@
...
+        add_subdirectory(ToolsCataMop/map_extractor)
+        add_subdirectory(ToolsCataMop/vmap_tools)
+        add_subdirectory(ToolsCataMop/mmaps_generator)
+    endif ()
+endif ()
 
-if(WIN32)
+if (WIN32)
     install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/ae_tools.bat DESTINATION ${ASCEMU_TOOLS_PATH})
 else()
```
Comment (Zyres) [inline src/tools/CMakeLists.txt:18]: missing

**Review by Zyres — CHANGES_REQUESTED** [review]
Hey :-) Please apply the rule to all controle statements (if, elseif, else, endif) otherwise we have to touch these files again.

File: cmake/Modules/FindMySQL.cmake (hunk 19 above)
Comment (schnek) [reply cmake/Modules/FindMySQL.cmake]: i updated) there was a problem in git

**Pull request merged**